src/package-hoister.js

```diff
@@ -291,8 +326,24 @@ export default class PackageHoister {
       }
     }
 
+    const peerDependencies = Object.keys(info.pkg.peerDependencies || {});
```

Comment:
May be we need a helper function, `getPeerDependencies` since I see this `Object.keys(pkg.peerDependencies || {})` in multiple places now.